doc/make.jl

```diff
@@ -16,6 +16,15 @@ end
 
 # Documenter Setup.
 
+# make links for stdlib package docs
+if Sys.iswindows()
```

Comment:
Will this dirty the repo if one builds the docs? Locally,  I just committed the symlinked file that is being created here.

Reply:
Will that work everywhere? Do we have any symlinks in the repo currently? I didn't want to be the first :)

Reply:
I've seen people do it for linking in the README.md into the documentation. Whether it will work everywhere... your guess  is much better than mine.